host/sgx/create.c

```diff
@@ -685,13 +685,29 @@ oe_result_t oe_create_enclave(
     {
         OE_RAISE(OE_FAILURE);
     }
-#if defined(__linux__)
 
-    /* Notify GDB that a new enclave is created */
-    oe_notify_gdb_enclave_creation(
-        enclave, enclave->path, (uint32_t)strlen(enclave->path));
+    {
+        oe_debug_enclave_t* debug_enclave =
```

Comment:
Should we check if we are in debug mode before creating the debug enclave? Also consider moving the added code to a separate function.

Reply:
Great suggestion.
I have added the check to not process an enclave if it is not a debug enclave.

There is a scenario that if there are release and debug enclaves loaded (for what ever reason), does the windows debugger plugin need to know the meta-data of the release enclave to safely capture a memory-dump. I have captured that in #2022